assets/styles.scss

```diff
@@ -37,3 +37,7 @@
     padding-left: 16px;
   }
 }
+
+.min-height-full {
+  min-height: 100vh;
```

Comment:
`min-height: 100%;` did not work as expected.

![image](https://user-images.githubusercontent.com/10208211/54254837-c1c88400-4598-11e9-87ef-4da0a548c771.png)